implementations/rust/ockam/ockam_command/src/reset/static/long_about.txt

```diff
@@ -0,0 +1,16 @@
+-----------------------------------------------------------------------
+WARNING:
+
+The `reset` command is dangerous and should be used with caution as it will remove all local
+state PERMANENTLY. This behaviour can be useful for cleaning up your development environment
+and starting afresh from scratch. However, you'll typically want to use specific delete commands such as
+`node delete`, `identity delete`, and `space delete` for more granular control on deletes.
+
+-----------------------------------------------------------------------
+
+By default, the `reset` command will only remove the local state (including Identities and Nodes).
+This is essentially the `database.sqlite3` file at `$OCKAM_HOME`. Spaces and Projects will only be deleted
```

Comment:
> This is essentially the `database.sqlite3` file at `$OCKAM_HOME`.

This is too detailed. The state can also be stored in a postgres database and not necessarily under the OCKAM_HOME directory. I'd delete it.